src/Compilers/Core/Portable/CodeAnalysisResources.Designer.cs

```diff
@@ -1343,7 +1343,7 @@ internal class CodeAnalysisResources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Windows PDB writer is not available -- could not find Microsoft.DiaSymReader.Native.{0}.dll.
+        ///   Looks up a localized string similar to Windows PDB writer doesn&apos;t support deterministic compilation: &apos;{0}&apos;.
```

Comment:
Ya there's a CPS issue that single file generators doesn't run in all the cases that the old PS did.